smartsim/_core/_cli/cli.py

```diff
@@ -59,12 +59,12 @@ def site(self):
 
     def dbcli(self):
         install_path = get_install_path()
-        script_path = install_path.joinpath("_core/bin/redis-cli").resolve()
+        script_path = next((install_path / "_core" / "bin").glob("*-cli")).resolve()
```

Comment:
I'm a bit wary of manipulating files with the regexes like this. Safer to be a bit more explicit even if it ends up adding code.